src/components/calendar/calendar.tsx

```diff
@@ -125,7 +125,7 @@ const Calendar: React.FC<CalendarProps> = ({
                 </div>
 
                 <span title={`${months[calendar.month]} ${calendar.year}`}>
-                  {months[calendar.month].slice(0, 3)} {calendar.year}
+                  {months[calendar.month].slice(0, 10)} {calendar.year}
```

Comment:
Unfortunately, we can't change this as it would impact every locale and everyone using the library. Please revert it.